wave_vision/tests/matcher_tests/brute_force_tests.cpp

```diff
@@ -0,0 +1,137 @@
+// C++ Headers
+#include <algorithm>
+
+// Libwave Headers
+#include "wave/wave_test.hpp"
+#include "wave/vision/detector/fast_detector.hpp"
+#include "wave/vision/descriptor/brisk_descriptor.hpp"
+#include "wave/vision/matcher/brute_force_matcher.hpp"
+
+namespace wave {
+
+const auto TEST_CONFIG = "tests/config/matcher/brute_force_matcher.yaml";
+const auto TEST_IMAGE_1 = "tests/data/image_center.png";
```

Comment:
I'll remove Lenna in a cleanup for the other two. There's a lot of good tips you've shown me for each one of these modules that I should go back and incorporate.